bpfd/src/command.rs

```diff
@@ -51,6 +50,50 @@ pub enum AttachType {
     SingleAttach(String),
 }
 
+#[derive(Debug, Serialize, Deserialize, Clone)]
+pub enum ImagePullPolicy {
+    Always = 0,
+    IfNotPresent = 1,
+    Never = 2,
+}
+
+impl TryFrom<i32> for ImagePullPolicy {
+    type Error = ParseError;
+
+    fn try_from(t: i32) -> Result<Self, Self::Error> {
+        let bpf_api_type = t.try_into()?;
+        match bpf_api_type {
+            bpfd_api::v1::ImagePullPolicy::Always => Ok(Self::Always),
+            bpfd_api::v1::ImagePullPolicy::IfNotPresent => Ok(Self::IfNotPresent),
+            bpfd_api::v1::ImagePullPolicy::Never => Ok(Self::Never),
+        }
+    }
+}
+
+#[derive(Debug, Serialize, Deserialize, Clone)]
+pub struct BytecodeImage {
```

Comment:
Move this and all nested types into oci-utils

Reply:
Make fields private and do everything internally 

Add constructors for the internal use case and the testing case, control all access through public functions